synapse/storage/events.py

```diff
@@ -56,7 +56,7 @@
 
 def encode_json(json_object):
     if USE_FROZEN_DICTS:
-        # ujson doesn't like frozen_dicts
+        # simplejson doesn't like frozen_dicts
```

Comment:
no

Reply:
D'oh, I thought i'd explicitly not changed that :/